components/ui/Alert.tsx

```diff
@@ -19,47 +19,143 @@
  */
 import * as classNames from 'classnames';
 import * as React from 'react';
+import styled from 'styled-components';
 import { translate } from '../../helpers/l10n';
 import Tooltip from '../controls/Tooltip';
 import AlertErrorIcon from '../icons/AlertErrorIcon';
 import AlertSuccessIcon from '../icons/AlertSuccessIcon';
 import AlertWarnIcon from '../icons/AlertWarnIcon';
 import InfoIcon from '../icons/InfoIcon';
-import ThemeContext from '../ThemeContext';
-import './Alert.css';
+import { SonarUiCommonThemeConsumer, Theme } from '../ThemeContext';
 
-type AlertDisplay = 'banner' | 'block' | 'inline';
+type AlertDisplay = 'banner' | 'inline' | 'block';
 type AlertVariant = 'error' | 'warning' | 'success' | 'info';
 
-export interface AlertProps {
+export interface Props {
```

Comment:
I believe we now prefix all exported `Props` interfaces with the component name, to make it easier to find them. You should keep it as `AlertProps`.

Reply:
I was not aware of such things. But that makes sense.